### PR TITLE
AC-2792::Common inputs lack autocomplete attributes (pattern)

### PR DESCRIPTION
**Pull request by glo71317**
## Description
Common inputs lack autocomplete attributes (pattern)

Locations (representative sample):

Checkout - Shipping Information
Checkout - Payment
1. Manually inspect markup for inputs in Guest Checkout form for autocomplete attributes.

**Actual Behavior**
The following common input fields lack a way to communicate purpose:

**Guest Checkout - Shipping Information form:**

Email
First Name
Country
Street Address
Street Address 2
City
State
Zip / Postal Code
Phone Number

**Checkout - Payment form:**

Cardholder Name
Card Number
Expiration Date
CVV
When an input field's purpose is not communicated (such as by use of the autocomplete attribute), users of assistive technology or users with cognitive disabilities can have trouble understanding a field's purpose. Users with limited mobility may have difficulty entering data that they commonly enter, such as address fields and phone numbers.

**Expected Behavior**
Ensure that common input fields communicate purpose – this is generally provided by the autocomplete values (HTML 5.3). Common fields are those included in this list: https://www.w3.org/TR/WCAG21/#input-purposes.

## Related Issue
Closes https://jira.corp.magento.com/browse/AC-2792.

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### Resolved issues:
1. [x] resolves magento/pwa-studio#3838: AC-2792::Common inputs lack autocomplete attributes (pattern)

### CODE REVIEW

File: packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/guestForm.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ const GuestForm = props => {
                         })}
                     >
                         <TextInput
-                            autoComplete="off"
+                            autoComplete="Shipping email"
```
Comment (chris-brabender) [inline packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/guestForm.js:148]: For internationalisation all strings should be translatable.

Comment (glo71317) [reply packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/guestForm.js]: done @chris-brabender please have a look

**Comment by supernova-at** [comment]
@magento create issue from PR

**Review by jasonrpoteet — APPROVED** [review]

**Review by jasonrpoteet — CHANGES_REQUESTED** [review]
Need to update unit tests to pass

**Comment by OneCricketeer** [comment]
The instance crashed which is why the tests time out.

**Comment by haal9000** [comment]
run pr-deploy

**Comment by haal9000** [comment]
The instance health went to severe as it was not able to find the PR image in ECR. Did a pr-deploy to build and deploy the PR image

**Review by anthoula — DISMISSED** [review]

**Comment by dpatil-magento** [comment]
run pr-deploy

**Comment by dpatil-magento** [comment]
run cypress

**Review by dpatil-magento — APPROVED** [review]

**Comment by dpatil-magento** [comment]
Cypress Failures are on `develop` not related to this pr.

**Comment by carpet-ray** [comment]
> ## Description
> 
> Common inputs lack autocomplete attributes (pattern)
> 
> Locations (representative sample):
> 
> Checkout - Shipping Information Checkout - Payment
> 
>     1. Manually inspect markup for inputs in Guest Checkout form for autocomplete attributes.
> 
> 
> **Actual Behavior** The following common input fields lack a way to communicate purpose:
> 
> **Guest Checkout - Shipping Information form:**
> 
> Email First Name Country Street Address Street Address 2 City State Zip / Postal Code Phone Number
> 
> **Checkout - Payment form:**
> 
> Cardholder Name Card Number Expiration Date CVV When an input field's purpose is not communicated (such as by use of the autocomplete attribute), users of assistive technology or users with cognitive disabilities can have trouble understanding a field's purpose. Users with limited mobility may have difficulty entering data that they commonly enter, such as address fields and phone numbers.
> 
> **Expected Behavior** Ensure that common input fields communicate purpose – this is generally provided by the autocomplete values (HTML 5.3). Common fields are those included in this list: https://www.w3.org/TR/WCAG21/#input-purposes.
> ## Related Issue
> 
> Closes https://jira.corp.magento.com/browse/AC-2792.
> ## Acceptance
> ### Verification Stakeholders
> ### Specification
> ## Verification Steps
> ## Checklist
> 
>     * I have added tests to cover my changes, if necessary.
> 
>     * I have added translations for new strings, if necessary.
> 
>     * I have updated the documentation accordingly, if necessary.
> 
> 
> ### Resolved issues:
> 
>     1. [x]  resolves [[Issue] AC-2792::Common inputs lack autocomplete attributes (pattern) #3838](https://github.com/magento/pwa-studio/issues/3838): AC-2792::Common inputs lack autocomplete attributes (pattern)

Hello, 
From what I see, the values of autocomplete attribute are not all valid. For example, the `autocomplete="First Name"` does not exist. The value should probably be `"given-name"` in this case. It's also a best practice to write these values in lower cases. 

It could be worth checking them to see if they are all valid. [https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill](url)

**Pull request merged**